plugins/tag_search/lib/model/index/TagIndex.php

```diff
@@ -43,7 +43,7 @@ public static function getIndexFieldsMap()
 		{
 			self::$fieldsMap = array(
 				'int_id' => 'intId',
-				'tag' => 'TagWithEqual',
+				'tag' => 'tag',
```

Comment:
This is how it was generated, but I now see that TagWithEqual is a real getter, I'll see if i can modify IndexSchema.xml to generate the old output.
There was a similar issue in entry index, that 2 fields were added manually without updating IndexSchema.xml